### PR TITLE
Update BN/NAT listing and Staff Log

### PR DESCRIPTION
**Pull request by zeusminus**
- Added [Scotty](https://osu.ppy.sh/users/11085809) to NAT members listing (osu!mania)

### CODE REVIEW

File: wiki/Staff_Log/2020/en.md [hunk 1]
```diff
@@ -328,4 +329,4 @@ Abbreviations for user groups are used throughout this log:
 - 2020-08-02: Removed [Icekalt](https://osu.ppy.sh/users/5410645) from **BN**
 - 2020-08-03: Removed [Rizen](https://osu.ppy.sh/users/3056732) from **BN**
```
Comment (hypercyte) [inline wiki/Staff_Log/2020/en.md:331]:
```suggestion

#### Removals

- 2020-08-07: Removed [Scotty](https://osu.ppy.sh/users/11085809) from **BN**

```

File: wiki/Staff_Log/2020/en.md [hunk 2]
```diff
@@ -318,6 +318,7 @@ Abbreviations for user groups are used throughout this log:
 
 - 2020-08-05: Added [Ayucchi](https://osu.ppy.sh/users/7823498) to **Probationary BN**
 - 2020-08-05: Added [Hobbes2](https://osu.ppy.sh/users/8157492) to **Probationary BN**
+- 2020-08-07: Added [Scotty](https://osu.ppy.sh/users/11085809) to **NAT**
```
Comment (hypercyte) [inline wiki/Staff_Log/2020/en.md:321]:
```suggestion
```

Comment (hypercyte) [reply wiki/Staff_Log/2020/en.md:321]: Don't know how to suggest changes above this point, but NAT additions should be outside the Beatmap Nominators section

Comment (zeusminus) [reply wiki/Staff_Log/2020/en.md]: thanks for pointing it out, fixed it

File: wiki/Staff_Log/2020/en.md [hunk 3]
```diff
@@ -328,4 +330,8 @@ Abbreviations for user groups are used throughout this log:
 - 2020-08-02: Removed [Icekalt](https://osu.ppy.sh/users/5410645) from **BN**
 - 2020-08-03: Removed [Rizen](https://osu.ppy.sh/users/3056732) from **BN**
 
-<!-- last update: 2020-08-05 17 UTC added hobbes2 to probationary bn -->
+#### Removals
```
Comment (hypercyte) [inline wiki/Staff_Log/2020/en.md:333]: oops we don't need the removals section, didn't notice one already existed

File: wiki/Staff_Log/2020/en.md [hunk 4]
```diff
@@ -327,5 +329,6 @@ Abbreviations for user groups are used throughout this log:
 
 - 2020-08-02: Removed [Icekalt](https://osu.ppy.sh/users/5410645) from **BN**
 - 2020-08-03: Removed [Rizen](https://osu.ppy.sh/users/3056732) from **BN**
+- 2020-08-07: Removed [Scotty](https://osu.ppy.sh/users/11085809) from **BN**
```
Comment (TPGPL) [inline wiki/Staff_Log/2020/en.md:332]:
```suggestion
```

File: wiki/Staff_Log/2020/en.md [hunk 5]
```diff
@@ -312,6 +312,8 @@ Abbreviations for user groups are used throughout this log:
 
 ## August
 
+- 2020-08-07: Added [Scotty](https://osu.ppy.sh/users/11085809) to **NAT**
```
Comment (TPGPL) [inline wiki/Staff_Log/2020/en.md:315]:
```suggestion
- 2020-08-07: Moved [Scotty](https://osu.ppy.sh/users/11085809) from **BN** to **NAT**
```

**Pull request merged**